nameservice/tutorial/16-module.md

```diff
@@ -8,128 +8,9 @@ The Cosmos SDK provides a standard interface for modules. This [`AppModule`](htt
 
 Start by opening two new files, `module.go` and `genesis.go`. We will implement the AppModule interface in `module.go` and the functions specific to genesis state management in `genesis.go`. The genesis-specific methods on your AppModule struct will be pass-though calls to those defined in `genesis.go`.
 
-Lets start with adding the following code to `module.go`. We will leave a number of the functions unimplemented for now.
+Lets start with adding the following code to `module.go`.	The scaffolding tool has already filled in all the functions with the data that is needed, but there are a couple todo's, you will have to add in the modules that this module depends on in the `AppModule` type.
```

Comment:
```suggestion
Lets start with adding the following code to `module.go`. The scaffolding tool has already filled in all the functions with the data that is needed but there are still a couple of to-dos. You will have to add the modules that this module depends on in the `AppModule` type.
```
Are there other to-dos or is it just this one thing?

Reply:
there are todos layed out throughout the scaffold template, but they get done while going through the tutorial, will add notes to remove the todos